Gemfile

```diff
@@ -16,10 +16,16 @@ gem 'sass-rails', '~> 5.0'
 gem 'turbolinks'
 gem 'uglifier', '>= 1.3.0'
 
-group :development do
+group :development, :test do
+  gem 'cucumber-rails', require: false
+  gem 'database_cleaner'
   gem 'pry-byebug'
   gem 'spring'
-  gem 'web-console', '~> 2.0'
+  
+end
+
+group :development do
+  gem 'web-console', '~> 2.0'  
```

Comment:
Is the addition of this gem really in the scope for this chore?
If it is then I would like to see it mentioned in the description of this PR

Reply:
This is an existing gem in existing group(earlier on line 22,in :development group). Since we modified the exisiting group we created a group below to move and retain that gem.

Reply:
Okay :+1: